taming/modules/vqvae/quantize.py

```diff
@@ -91,12 +91,13 @@ def forward(self, z):
 
     def get_codebook_entry(self, indices, shape):
         # shape specifying (batch, height, width, channel)
```

Comment:
Could you give a little more context why we need this? we generally use VectorQuantizer2 not VectorQuantizer.

Reply:
retraced to the original code